index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;1,100;1,200&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,100;0,200;0,300;1,100;1,200&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <header id="toolbar">
+        <div class="name_div"><p>Soulemanou Babanou</p></div>
+        <div class="menu_icon"><img src="Resources/Images/Icon-Menu20.png" alt=""></div>
+    </header>
+    <section id="headlines">
+        <div class="headline_texts">
+            <h1>Hey There.<br>I'm Babanou</h1>
+            <h3>I'am a Software Developer</h3>
+            <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,don’t hestiate to contact me.</p>
+            <div class="social_media">
+                <img src="Resources/Images/Vector.png" alt="Github-logo">
+                <img src="Resources/Images/Icon-Linkedin.png" alt="linkin-logo">
+                <img src="Resources/Images/Icon-Angelist.png" alt="angelist-logo">
+                <img src="Resources/Images/Icon-Twitter.png" alt="twitter-logo">
+                <img src="Resources/Images/Icon-Medium.png" alt="medium-logo">
+            </div>
+        </div>
+            <img src="Resources/Images/illustration-header-mobile.png" alt="illustration-png" id="illustration-png">
```

Comment:
- [ ] You seem to have used an `img` tag here to add the decoration in the background. Decorations are always background images so kindly remove this `img` tag and add this image as a [`background`](https://www.edureka.co/blog/background-image-in-html/#:~:text=The%20most%20common%20%26%20simple%20way,inside%20the%20tag.&text=The%20background%20attribute%20which%20we,background%20image%20in%20a%20webpage.) property to your headline section. You can use the `background-position` attribute to position it correctly and `background-size` to size it.

Reply:
Wow, I like the changes that I am seeing here thanks a lot.